tools/kotlin-native-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/plugin/experimental/internal/KotlinNativeVariantIdentity.kt

```diff
@@ -0,0 +1,31 @@
+package org.jetbrains.kotlin.gradle.plugin.experimental.internal
+
+import org.gradle.api.internal.component.UsageContext
+import org.gradle.api.model.ObjectFactory
+import org.gradle.api.provider.Provider
+import org.gradle.language.cpp.internal.NativeVariantIdentity
+import org.jetbrains.kotlin.konan.target.KonanTarget
+
+open class KotlinNativeVariantIdentity(name: String,
```

Comment:
According to the kotlin style guide (see link above) each property should be in a new line